### PR TITLE
[TA-1250] Handle inactive accounts

### PR DESCRIPTION
**Pull request by GuillemGarciaDev**
# [TA-1250] Handle inactive accounts

## Tasks

- [[TA-1250] Handle inactive accounts](https://www.notion.so/Handle-inactive-accounts-d1143b4c627c4a78857e187d829a260d?pvs=4)

## Changes

- `useIsActiveAccount` query to check if an account is activated or not
- `SignatureScaffold` handles account activation
- renaming on `SignerModalContext` (due to usage on both modals)
- locale updates

IMG_0610

### CODE REVIEW

File: src/module/signer/queries/useIsActiveAccount.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    const { serviceInstance, index, network } = useServiceInstance(walletIndex);
+
+    return useQuery(
+        [Queries.ACCOUNT_IS_ACTIVE, index, network],
+        async () => {
+            const accountId = await serviceInstance.getAccountId();
+            return await serviceInstance.accountExists(accountId);
+        },
+        {
+            enabled: !!serviceInstance,
```
Comment (AgustinMJ) [inline src/module/signer/queries/useIsActiveAccount.ts:15]: Important el `cacheTime` i `staleTime` a 0 per no cachejar aquesta call

File: src/module/signer/components/display/SignMessageDetails/SignMessageDetails.tsx [hunk 2]
```diff
@@ -19,6 +20,7 @@ const SignMessageDetails = ({ receiver, message, metadata }: SignMessageDetailsP
             <Container>
                 <ActionDetailField label={translate("message")} description={message} />
             </Container>
+            <SignerWalletSelector />
```
Comment (AgustinMJ) [inline src/module/signer/components/display/SignMessageDetails/SignMessageDetails.tsx:23]: Aquí s'ha de ficar `SignerWalletSelector`? No passa res si no està activa la conta?

**Review by AgustinMJ — CHANGES_REQUESTED** [review]

File: src/module/signer/components/display/SignMessageDetails/SignMessageDetails.tsx (hunk 2 above)
Comment (GuillemGarciaDev) [reply src/module/signer/components/display/SignMessageDetails/SignMessageDetails.tsx:23]: IMG_0611

Comment (GuillemGarciaDev) [reply src/module/signer/components/display/SignMessageDetails/SignMessageDetails.tsx:23]: No passa res perque ho handleja el `SignatureScaffold` 👍🏻

**Review by AgustinMJ — APPROVED** [review]

**Pull request merged**